### DISCUSSION TITLE
Epic MegaGrants

### DISCUSSION THREAD

**Discussion by rmerkushin** [nxrighthere/UnrealCLR#7] [Legacy]
Hi man! I think you should apply on [Epic MegaGrants](https://www.unrealengine.com/en-US/megagrants)!

P.S.: Anyway, thanks for your work!

**Comment by nxrighthere** [comment]
Hey, thanks! The project was submitted, and Epic is in touch with me. 👌

**Comment by jimboc1** [comment]
> Hey, thanks! The project was submitted, and Epic is in touch with me. 👌

First thing I said to @Schotime when he introduced UnrealCLR to me was "MegaGrant written all over it - so much c# talent out there resorting to Unity."

**Comment by nxrighthere** [comment]
@jimboc1 Thanks! We are moving forward. ❤️

**Comment by nxrighthere** [comment]
[We got it](https://twitter.com/nx_righthere/status/1337336362522079232)! ❤️

**Reply by MostHated** [reply]
Awesome man, glad to hear! To be completely honest, though, I am not really surprised. Seemed like a 'shoo-in'. Any plans for the funds? I am guessing its probably not enough to hire any help full time or anything?

**Reply by nxrighthere** [reply]
We have a long-term plan for at least the next three years, and this is something that we discussed with Epic. More exciting news will be announced next year!

**Reply by siennathesane** [reply]
That's super exciting!

**Reply by dagillespie** [reply]
fingers crossed you make it to UE5!

**Comment by spo9kor** [comment]
Epic finally sponsoring c#. too late but it's happening.  thank you to all supporters for empowering c# community. 

long live UnrealCLR.

**Reply by nxrighthere** [reply]
Thank you!

**Reply by darkguy2008** [reply]
Yeah finally! I'm tired of dealing with all the Unity crap but it's better than dealing with C++... Definitely looking forward to see UnrealCLR as an official C# support for Unreal, or something. Congratulations!
